tests/oracle/test-oracle.cpp

```diff
@@ -180,39 +180,68 @@ struct basic_table_creator : public table_creator_base
     }
 };
 
-TEST_CASE("Oracle nested statement", "[oracle][blob]")
+TEST_CASE("Oracle nested statement", "[oracle][cursors]")
```

Comment:
Is the change to this test case really related to the main topic of the PR?

Reply:
No, this addition is an "incidental correction" )))